src/Template/Bake/Model/entity.ctp

```diff
@@ -20,6 +20,9 @@ use Cake\ORM\Entity;
 
 /**
  * <%= $name %> Entity.
+<% foreach ($fields as $field): %>
```

Comment:
Using `$fields` won't work properly as its purpose was to hold the mass assignable fields for the `$_accessible` property, it excludes `created`, `modified`, `updated` and primary key fields, and respects the (**[now kinda superflous](https://github.com/cakephp/bake/pull/134)** but still existent) [**`no-fields`**](https://github.com/cakephp/bake/blob/1.0.11/src/Shell/Task/ModelTask.php#L877) and [**`fields`**](https://github.com/cakephp/bake/blob/1.0.11/src/Shell/Task/ModelTask.php#L880) options, ie the `$fields` variable will not include all possible fields, and might even be empty.

I think either `$fields` should be used again to define the accessible fields, but this time only in case the `fields` or `no-fields` option was used, or the options should be removed and **[`$fields` should be changed](https://github.com/cakephp/bake/blob/1.0.11/src/Shell/Task/ModelTask.php#L438)** to hold _all_ existing fields.